lib/foreman_tasks/tasks/export_tasks.rake

```diff
@@ -255,6 +257,16 @@ namespace :foreman_tasks do
         tasks.each_with_index do |task, count|
           File.open(File.join(tmp_dir, "#{task.id}.html"), 'w') { |file| file.write(PageHelper.pagify(renderer.render_task(task))) }
           puts "#{count + 1}/#{total}"
+        rescue StandardError => e
+          puts "WARNING: task failed to export. Additional task details below."
+          puts task.inspect
+          if skip_errors?
```

Comment:
the variable is just `skip_errors` without the question mark

Reply:
resolved.